meta-refkit/conf/distro/refkit.conf

```diff
@@ -74,6 +74,13 @@ PACKAGECONFIG_remove_pn-pulseaudio = "avahi"
 # leave out readline, gdbm, and db
 PACKAGECONFIG_pn-python3 = ""
 
+# drop python2 support
+PACKAGECONFIG_remove_pn-opkg-utils = "python"
+PACKAGECONFIG_remove_pn-libxml2 = "python"
+# there's a bug in bitbake that causes the build-deps QA check to check an empty package
```

Comment:
Yocto is such a nice thing I'm still struggling with how to formulate the bug properly. So far I've come up with this:

----
Given `localdata` is a local copy of `d` after calling `localdata.setVar('OVERRIDES', 'libxml2-python')` its method `getVar()` starts to return a whitespace (" ") for `localdata.getVar('RDEPENDS')` if `libxml2-python` is made an empty package with `PACKAGECONFIG_remove = "python"` in a bbappend

But if `libxml2-python` is made an empty package with `PACKAGECONFIG_remove_pn-libxml2 = "python"` in a distro config then `localdata.getVar('RDEPENDS')` returns " python-core"
 and that breaks the 'build-deps' QA check because it thinks that `libxml2-python` actually exists even if it's empty.

----

Can you get it?